python/pyspark/sql/streaming/stateful_processor.py

```diff
@@ -420,10 +411,27 @@ def handleInputRows(
         timer_values: TimerValues
                       Timer value for the current batch that process the input rows.
                       Users can get the processing or event time timestamp from TimerValues.
+        """
+        return iter([])
+
+    def handleExpiredTimer(
```

Comment:
Just double check that this method is not required for users to implement, correct?